ts/Core/Chart/GanttChart.ts

```diff
@@ -147,27 +146,29 @@ class GanttChart extends Chart {
                 defaultLinkedTo = 0;
             }
             return merge(
-                defaultOptions.xAxis,
-                { // defaults
+                // Defaults
```

Comment:
Not sure, if this should have defaultOptions.xAxis restored as well, for similar cases like #3802.

Reply:
Strictly speaking, yes. Nice catch! But on the contrary to Stock charts, this one never worked: https://jsfiddle.net/0L3fvyhb/ . I'll see if we can resolve it the same way as for Stock.